src/Microsoft.DotNet.Build.CloudTestTasks/CreateAzureContainer.cs

```diff
@@ -3,110 +3,168 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Globalization;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using System.Net.Http;
 using Microsoft.Build.Framework;
-using Microsoft.Build.Utilities;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Blob;
+
+using Task = Microsoft.Build.Utilities.Task;
 
 namespace Microsoft.DotNet.Build.CloudTestTasks
 {
+
     public sealed class CreateAzureContainer : Task
     {
         /// <summary>
-        /// The Azure account name used when creating the connection string.
+        ///     The Azure account key used when creating the connection string.
```

Comment:
Why are there all of these leading white spaces in the comments now?  They are making the diff incredibly noisy.